.github/workflows/swift-toolchain.yml

```diff
@@ -2282,6 +2285,13 @@ jobs:
           $CLANG_CL = cygpath -m ${{ github.workspace }}/BinaryCache/Library/Developer/Toolchains/unknown-Asserts-development.xctoolchain/usr/bin/clang-cl.exe
           $SWIFTC = cygpath -m ${{ github.workspace }}/BinaryCache/Library/Developer/Toolchains/unknown-Asserts-development.xctoolchain/usr/bin/swiftc.exe
 
+          # Worarkound CMake 3.30 issue where CMAKE_[*]_FLAGS are passed as-is to the linker driver.
+          # TODO: Once we have CMake 4.0, set CMP0181 to NEW and pass these as "LINKER:" flags.
+          $CMAKE_SHARED_LINKER_FLAGS =
+            ("${{ matrix.shared_linker_flags }}".Split(" ").Where({ $_.Trim() -ne ''}) | ForEach-Object { "-Xlinker $_" }) -join " "
```

Comment:
What happens if there is a space in a path?

Reply:
It will break, but it also breaks currently too as CMake also parses the space as a separator. Note that we don't perform any kind of escape for quotes here either.
The good news is that when we switch to CMake 4.0, we'll be able to handle these properly since `LINKER:` supports comma-separated values.

Reply:
Hmm, I thought it was possible to get that passed properly. Well, I guess if it's no worse off ...